lib/Graphs/PAG.cpp

```diff
@@ -58,6 +97,7 @@ const std::string ValPN::toString() const {
     std::string str;
     raw_string_ostream rawstr(str);
     rawstr << "ValPN ID: " << getId();
+    rawstr << "\n";
```

Comment:
How about making it only one line? In case If someone would like to debug in the terminal

Reply:
May I know why you add `\n` here? Keep it one line would be good for debugging in terminal.

Reply:
The new-line is needed to make the .dot output look nicer (lines not too long).

Reply:
But the first line will be too short. Can we make it as one line for now? You can add brackets `[]` for nodeID or for instruction to make them more distinguishable?

Reply:
OK. I'm making it one line (in my next push).